index.html

```diff
@@ -1,15 +1,86 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Segunda-Tarea</title>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
+    <title>Segunda-Tarea</title>
+	<title>Mi Tienda Online</title>
 </head>
 <body>
-  <header></header>
-  <main></main>
-  <footer></footer>
-</body>
+    <header class="header">
+        <div class="header__top-top">
+          <div class="header__top container">
+            <div class="header__logo">
+              <img src="image/logo.png" alt="Logo">
+            </div>
+            <div class="header__search-bar">
+              <input type="text" placeholder="¿Qué buscas?">
+              <button>
+                <i class="fas fa-search">
+                  <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M7 12C9.76142 12 12 9.76142 12 7C12 4.23858 9.76142 2 7 2C4.23858 2 2 4.23858 2 7C2 9.76142 4.23858 12 7 12Z" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                    <path d="M14.0001 14L10.5334 10.5333" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                  </svg>
+                </i>
```

Comment:
Ese tag `i` no hace nada aquí
```suggestion
                <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
                  <path fill-rule="evenodd" clip-rule="evenodd" d="M7 12C9.76142 12 12 9.76142 12 7C12 4.23858 9.76142 2 7 2C4.23858 2 2 4.23858 2 7C2 9.76142 4.23858 12 7 12Z" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
                  <path d="M14.0001 14L10.5334 10.5333" stroke="#EEEEF7" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
                </svg>
```